shared/chat/conversations-list/index.native.js

```diff
@@ -1,3 +1,231 @@
 // @flow
-const ConversationList = () => null
+import React from 'react'
+import {Text, MultiAvatar, Icon, /* Usernames, Markdown, */ Box, ClickableBox, NativeScrollView} from '../../common-adapters/index.native'
+import {globalStyles, globalColors, globalMargins} from '../../styles'
+import {shouldUpdate} from 'recompose'
+
+import type {Props, RowProps} from './'
+
+const AddNewRow = ({onNewChat}: {onNewChat: () => void}) => (
+  <Box
+    style={{...globalStyles.flexBoxRow, alignItems: 'center', flexShrink: 0, justifyContent: 'center', minHeight: 48}}>
+    <ClickableBox style={{...globalStyles.flexBoxColumn}} onClick={onNewChat}>
+      <Box style={{...globalStyles.flexBoxRow, alignItems: 'center', justifyContent: 'center'}}>
+        <Icon type='iconfont-new' style={{color: globalColors.blue, marginRight: 9}} />
+        <Text type='BodyBigLink'>New chat</Text>
+      </Box>
+    </ClickableBox>
+  </Box>
+)
+
+// All this complexity isn't great but the current implementation of avatar forces us to juggle all these colors and
+// forces us to explicitly choose undefined/the background/ etc. This can be cleaned up when avatar is simplified
+function rowBorderColor (idx: number, isLastParticipant: boolean, backgroundColor: string) {
+  // Only color the foreground items
+  if (isLastParticipant) {
+    return undefined
+  }
+
+  // We don't want a border if we're a single avatar
+  return !idx && isLastParticipant ? undefined : backgroundColor
+}
+
+const Avatars = ({participants, youNeedToRekey, participantNeedToRekey, isMuted, hasUnread, isSelected, backgroundColor}) => {
+  const avatarCount = Math.min(2, participants.count())
+
+  let icon
+  if (isMuted) {
+    icon = <Icon type={isSelected ? 'icon-shh-active-16' : 'icon-shh-16'} style={avatarMutedIconStyle} />
+  } else if (participantNeedToRekey || youNeedToRekey) {
+    icon = <Icon type={isSelected ? 'icon-chat-addon-lock-active-8' : 'icon-chat-addon-lock-8'} style={avatarLockIconStyle} />
+  }
+
+  const avatarProps = participants.slice(0, 2).map((username, idx) => ({
+    backgroundColor,
+    borderColor: rowBorderColor(idx, idx === (avatarCount - 1), backgroundColor),
+    size: 24,
+    style: {
+      opacity: youNeedToRekey || participantNeedToRekey ? 0.4 : 1,
+    },
+    username,
+  })).toArray()
+
+  return (
+    <Box style={{...globalStyles.flexBoxRow, alignItems: 'center', flex: 1, justifyContent: 'flex-start', maxWidth: 48, paddingLeft: 4}}>
+      <MultiAvatar singleSize={32} multiSize={24} avatarProps={avatarProps} />
+      {icon}
+    </Box>
+  )
+}
+
+const TopLine = ({hasUnread, showBold, participants, subColor, timestamp, usernameColor, commaColor}) => {
+  return null
+  // const boldOverride = showBold ? globalStyles.fontBold : null
```

Comment:
this is just so i can get this merged so other code can move forward. its already implemented in my next branch